desktop/apps/auction2/components/page/Header.js

```diff
@@ -2,57 +2,86 @@ import AddToCalendarView from 'desktop/components/add_to_calendar/react'
 import PropTypes from 'prop-types'
 import React from 'react'
 import Registration from './Registration'
+import { connect } from 'react-redux'
 
-export default function Header (props) {
-  const { auction } = props
-  const showAddToCalendar = !(auction.isClosed() || auction.isLiveOpen())
+function Header (props) {
+  const {
+    description,
+    isAuctionPromo,
+    liveStartAt,
+    name,
+    showAddToCalendar,
+    upcomingLabel
+  } = props
```

Comment:
Would it make sense to destructure these in the function declaration, even if its multi-line? Like

```
function Header ({
  description,
  isAuctionPromo,
  // ...
}) {
  // ...
}
```

It looks a little funky, but then again, so does everything! I do favor the concept of not giving names to things that aren't needed more than once.